app/controllers/merchant/orders_controller.rb

```diff
@@ -5,4 +5,16 @@ def show
     @order = Order.find(params[:order_id])
     @item_orders = @order.find_order(current_user.merchant_id)
   end
+
+  def update_item
+    item_order = ItemOrder.find(params[:item_order_id])
+    item_order.update_attributes(status: 1)
+    item_order.update_item_inventory
+    flash[:success] = "The #{item_order.item.name} is now fulfilled."
+    if item_order.order.all_items_fulfilled?
+      item_order.order.update_status(1)
+    end
+
```

Comment:
@cjkelling and @CoopTang, @freeheeling and I would like your opinion on the update_item action. (#15 ) just says when all items in an order have been 'fulfilled' the order status changes from 'pending' to 'packaged'. Our line of thought was some action needed to take place to prompt the changing of the order status. That action would likely be the last unfulfilled item in the order being fulfilled. Open to other suggestions and feedback.

Reply:
I think you are on the right track. We need something that checks the status of all ItemOrders and verifies the status. We check all the status from the order and make sure they equal fulfilled then we change the whole order status. Looks good to me.

Reply:
I also have to agree. There needs to be some sort of event that happens after that last item has been fulfilled.

Reply:
@CoopTang and @cjkelling, thank you both! Once you are done reviewing the rest of the pull request and are good with the changes can you please merge?

Reply:
@lrs8810 @freeheeling @cjkelling, shouldn't the user_order_controller be in charge of  subtracting the item quantity from the inventory (i.e. when an order is placed/created)? This would kind of be following the same line of thought as #16 where when a user cancels an order, then the user_order_controller returns the inventory.

Reply:
@CoopTang my line of thinking is the item order quantity is only subtracted when the merchant user clicks 'fulfill order' so it's a merchant action, not a user action that subtracts the quantity from the inventory. Happy to discuss further.